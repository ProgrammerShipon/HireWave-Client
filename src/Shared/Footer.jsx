import { Link } from "react-router-dom";

// react icons
import {
  FaFacebookF,
  FaInstagram,
  FaLinkedin,
  FaTwitter,
} from "react-icons/fa";

// logo
import logo from "../Assets/images/logo-01.png";

const Footer = () => {
  return (
    <footer className="text-dark pt-10">
      <div className="container">
        <div className="grid grid-cols-1 md:grid-cols-3 lg:grid-cols-6 gap-9 pt-12">
          {/* Logo, address, and social media links */}
          <div className="col-span-1 md:col-span-2">
            {/* logo */}
            <Link to="/">
              <img className="w-56" src={logo} alt="hire wave logo" />
            </Link>

            <p className="mt-5 max-w-xs">
              Our platform is more than just a hiring agency. it's a dynamic
              ecosystem where talent meets opportunity, innovation meets
              collaboration, and dreams meet reality.{" "}
            </p>

            {/* social icons */}
            <div className="flex items-center gap-2 mt-8">
              <Link
                to="/"
                className="text-green h-9 w-9 flex items-center justify-center rounded-lg border border-green shadow-lg shadow-green/20 duration-500 ease-in-out hover:rounded-[100%]"
              >
                <FaFacebookF size="20px" />
              </Link>

              <Link
                to="/"
                className="text-green h-9 w-9 flex items-center justify-center rounded-lg border border-green shadow-lg shadow-green/20 duration-500 ease-in-out hover:rounded-[100%]"
              >
                <FaTwitter size="20px" />
              </Link>

              <Link
                to="/"
                className="text-green h-9 w-9 flex items-center justify-center rounded-lg border border-green shadow-lg shadow-green/20 duration-500 ease-in-out hover:rounded-[100%]"
              >
                <FaLinkedin size="20px" />
              </Link>

              <Link
                to="/"
                className="text-green h-9 w-9 flex items-center justify-center rounded-lg border border-green shadow-lg shadow-green/20 duration-500 ease-in-out hover:rounded-[100%]"
              >
                <FaInstagram size="20px" />
              </Link>
            </div>
          </div>

          {/* Job seekers */}
          <div>
            <h3 className="text-2xl font-medium text-dark capitalize drop-shadow-lg">
              Job seekers
            </h3>
            <ul className="flex flex-col gap-1 mt-5 text-gray">
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Browse Jobs
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Salary Tools
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Career Advice
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Student Career Center
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Resume writing services
                </Link>
              </li>
            </ul>
          </div>

          {/* Employers */}
          <div>
            <h3 className="text-2xl font-medium text-dark capitalize drop-shadow-lg">
              Employers
            </h3>
            <ul className="flex flex-col gap-1 mt-5 text-gray">
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  How to Hire
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Talent Marketplace
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Hire an Agency
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Hire worldwide
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Direct Contracts
                </Link>
              </li>
            </ul>
          </div>

          {/* Resources */}
          <div>
            <h3 className="text-2xl font-medium text-dark capitalize drop-shadow-lg">
              Resources
            </h3>
            <ul className="flex flex-col gap-1 mt-5 text-gray">
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Help and Support
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Success Stories
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Blogs
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Researches
                </Link>
              </li>
              <li>
                <Link to="/" className="hover:text-green duration-300">
                  Community
                </Link>
              </li>
            </ul>
          </div>

          {/* contact */}
          <div>
            <h3 className="text-2xl font-medium text-dark capitalize drop-shadow-lg">
              Contact Us
            </h3>
            <div className="text-gray">
              <p className="mt-5">
                43/7 Northern Tower, Banani, Dhaka-1263, Bangladesh
              </p>
              <p className="mt-4 lowercase">support@hirewave.com</p>
              <p>+8801543653473</p>
            </div>
          </div>
        </div>
      </div>

      {/* copy write */}
      <div className="border-t border-green mt-14 shadow-3xl">
        <div className="container">
          <div className="flex flex-col md:flex-row items-center justify-between py-5">
            <p>Â© 2023 Hire Wave. All rights reserved.</p>
            <div className="flex items-center gap-1">
              <Link to="/" className="hover:text-green duration-300">
                Privacy Policy
              </Link>{" "}
              |
              <Link to="/" className="hover:text-green duration-300">
                Terms of Services
              </Link>
            </div>
          </div>
        </div>
      </div>
    </footer>
  );
};

export default Footer;
