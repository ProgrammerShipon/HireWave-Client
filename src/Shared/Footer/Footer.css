li a:hover, .footer-items:hover{
    text-decoration: underline;
    color: rgb(4 120 87);
}

.footer-dropdown h1{
    letter-spacing: 2px;
}