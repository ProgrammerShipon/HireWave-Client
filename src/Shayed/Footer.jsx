const Footer = () => {
   return (
     <footer>
       <h1 className="text-3xl text-pink-700"> This is Footer Route </h1>
     </footer>
   );
};

export default Footer;