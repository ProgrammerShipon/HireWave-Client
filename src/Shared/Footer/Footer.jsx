import logo from '/Logo/black bg banner.png'
import './Footer.css'
import { FaFacebookF } from 'react-icons/fa';
import { BiLogoTelegram } from 'react-icons/bi';
import { FiTwitter, FiFacebook } from 'react-icons/fi';
import { AiOutlineYoutube } from 'react-icons/ai';
const Footer = () => {
  return (
    <footer className='text-white bg-green mt-10'>
      <div className='container py-5 md:flex justify-between gap-5 mt-0'>
        <div>
          <img className='w-1/2 ' src={logo} alt="" />
          <p className='mt-5 mb-2'>43/7 Northern Tower, Banani, Dhaka-1263</p>
          <p className='font-semibold'>+8801543653473</p>

          <div className='mt-4 md:mt-8'>
            <h1 className='mb-2'>Follow Us:</h1>
            <div className='flex gap-2'>
              <FiFacebook className='text-4xl p-1 rounded-full border-2 border-white text-white hover:bg-emerald-300' />
              <FiTwitter className='text-4xl p-1 rounded-full border-2 border-white text-white hover:bg-emerald-300' />
              <AiOutlineYoutube className='text-4xl p-1 rounded-full border-2 border-white text-white hover:bg-emerald-300' />
              <BiLogoTelegram className='text-4xl p-1 rounded-full border-2 border-white text-white hover:bg-emerald-300' />
              <FaFacebookF className='text-4xl p-1 rounded-full border-2 border-white text-white hover:bg-emerald-300' />
            </div>

          </div>
        </div>

        <section className='grid grid-cols-2 lg:grid-cols-4 gap-10 md:mt-0 mt-10'>
          <div>
            <h1 className='font-semibold text-emerald-700 mb-3'>Job seekers</h1>
            <ul>
              <li className='mb-2'><a href="">Browse Jobs</a></li>
              <li className='mb-2'><a href="">Salary Tools</a></li>
              <li className='mb-2'><a href="">Career Advice</a></li>
              <li className='mb-2'><a href="">Student Career Center</a></li>
              <li className='mb-2'><a href="">Resume writing services</a></li>
              <li className='mb-2'><a href="">Direct Contracts</a></li>
            </ul>
          </div>
          <div>
            <h1 className='font-semibold text-emerald-700 mb-3'>Employers</h1>
            <ul>
              <li className='mb-2'><a href="">How to Hire</a></li>
              <li className='mb-2'><a href="">Talent Marketplace</a></li>
              <li className='mb-2'><a href="">Hire an Agency</a></li>
              <li className='mb-2'><a href="">Hire worldwide</a></li>
              <li className='mb-2'><a href="">Direct Contracts</a></li>
            </ul>
          </div>
          <div>
            <h1 className='font-semibold text-emerald-700 mb-3'>Resources</h1>
            <ul>
              <li className='mb-2'><a href="">Help and Support</a></li>
              <li className='mb-2'><a href="">Success Stories</a></li>
              <li className='mb-2'><a href="">Blogs</a></li>
              <li className='mb-2'><a href="">Researches</a></li>
              <li className='mb-2'><a href="">Community</a></li>
              <li className='mb-2'><a href="">Events and Webinars</a></li>
              <li className='mb-2'><a href="">News and press</a></li>
            </ul>
          </div>
          <div>
            <h1 className='font-semibold text-emerald-700 mb-3'>Company</h1>
            <ul>
              <li className='mb-2'><a href="">About Us</a></li>
              <li className='mb-2'><a href="">Leadership</a></li>
              <li className='mb-2'><a href="">Careers</a></li>
              <li className='mb-2'><a href="">Our Impact</a></li>
              <li className='mb-2'><a href="">Contact Us</a></li>
            </ul>
          </div>
        </section>
      </div>

      <div className='md:flex justify-between container mt-10 pb-3 text-center'>
        <div>
          <p>Â© 2023 HireWave. All rights reserved.</p>
        </div>
        <div>
          <p><span className='footer-items'>Privacy Policy</span> | <span className='footer-items'>Terms of Services</span></p>
        </div>
        <div className='hidden md:block'>
          Website by <span className='footer-items'>Hiqmah-607</span>
        </div>
      </div>

    </footer>
  );
};

export default Footer;