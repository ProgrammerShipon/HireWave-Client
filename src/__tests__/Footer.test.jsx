import TestRenderer from 'react-test-renderer';
import { MemoryRouter } from "react-router";
import Footer from "../Shared/Footer";

describe('Footer.jsx', () => {
    it('test the children inside the footer component', () => {
        const footer = TestRenderer.create(<MemoryRouter><Footer /></MemoryRouter>).toJSON()
        expect(footer).toMatchSnapshot();
    })

    it("renders the logo", () => {
        const component = TestRenderer.create(
            <MemoryRouter>
                <Footer />
            </MemoryRouter>
        );
        const logo = component.root.findByType("img");
        expect(logo.props.alt).toBe("hire wave logo");
    });

    it("renders social media links", () => {
        const component = TestRenderer.create(
            <MemoryRouter>
                <Footer />
            </MemoryRouter>
        );
        const socialLinks = component.root.findAllByType("a");
        expect(socialLinks).toHaveLength(27);
    });

    it("renders candidate links", () => {
        const component = TestRenderer.create(
            <MemoryRouter>
                <Footer />
            </MemoryRouter>
        );

        // Use a data-testid to uniquely identify the link.
        const candidateLink = component.root.findByProps({
            "data-testid": "candidate-link",
        });

        expect(candidateLink.props.children).toBe("Browse Jobs");
    });
})