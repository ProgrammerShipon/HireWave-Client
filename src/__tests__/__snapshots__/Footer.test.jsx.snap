// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer.jsx > test the children inside the footer component 1`] = `
<footer
  className="pt-10 text-dark"
>
  <div
    className="container"
  >
    <div
      className="grid grid-cols-1 md:grid-cols-3 lg:grid-cols-6 gap-9 pt-12 "
    >
      <div
        className="order-first col-span-1 md:col-span-2"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <img
            alt="hire wave logo"
            className="w-56"
            src="/src/Assets/images/logo-01.png"
          />
        </a>
        <p
          className="max-w-xs mt-5"
        >
          Our platform is more than just a hiring agency. it's a dynamic ecosystem where talent meets opportunity, innovation meets collaboration, and dreams meet reality.
           
        </p>
        <div
          className="flex items-center gap-2 mt-8"
        >
          <a
            className="text-green h-9 w-9 flex items-center justify-center rounded-lg border border-green shadow-lg shadow-green/20 duration-500 ease-in-out hover:rounded-[100%]"
            href="/"
            onClick={[Function]}
          >
            <svg
              fill="currentColor"
              height="20px"
              stroke="currentColor"
              strokeWidth="0"
              style={
                {
                  "color": undefined,
                }
              }
              viewBox="0 0 320 512"
              width="20px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
              />
            </svg>
          </a>
          <a
            className="text-green h-9 w-9 flex items-center justify-center rounded-lg border border-green shadow-lg shadow-green/20 duration-500 ease-in-out hover:rounded-[100%]"
            href="/"
            onClick={[Function]}
          >
            <svg
              fill="currentColor"
              height="20px"
              stroke="currentColor"
              strokeWidth="0"
              style={
                {
                  "color": undefined,
                }
              }
              viewBox="0 0 512 512"
              width="20px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
              />
            </svg>
          </a>
          <a
            className="text-green h-9 w-9 flex items-center justify-center rounded-lg border border-green shadow-lg shadow-green/20 duration-500 ease-in-out hover:rounded-[100%]"
            href="/"
            onClick={[Function]}
          >
            <svg
              fill="currentColor"
              height="20px"
              stroke="currentColor"
              strokeWidth="0"
              style={
                {
                  "color": undefined,
                }
              }
              viewBox="0 0 448 512"
              width="20px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M416 32H31.9C14.3 32 0 46.5 0 64.3v383.4C0 465.5 14.3 480 31.9 480H416c17.6 0 32-14.5 32-32.3V64.3c0-17.8-14.4-32.3-32-32.3zM135.4 416H69V202.2h66.5V416zm-33.2-243c-21.3 0-38.5-17.3-38.5-38.5S80.9 96 102.2 96c21.2 0 38.5 17.3 38.5 38.5 0 21.3-17.2 38.5-38.5 38.5zm282.1 243h-66.4V312c0-24.8-.5-56.7-34.5-56.7-34.6 0-39.9 27-39.9 54.9V416h-66.4V202.2h63.7v29.2h.9c8.9-16.8 30.6-34.5 62.9-34.5 67.2 0 79.7 44.3 79.7 101.9V416z"
              />
            </svg>
          </a>
          <a
            className="text-green h-9 w-9 flex items-center justify-center rounded-lg border border-green shadow-lg shadow-green/20 duration-500 ease-in-out hover:rounded-[100%]"
            href="/"
            onClick={[Function]}
          >
            <svg
              fill="currentColor"
              height="20px"
              stroke="currentColor"
              strokeWidth="0"
              style={
                {
                  "color": undefined,
                }
              }
              viewBox="0 0 448 512"
              width="20px"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M224.1 141c-63.6 0-114.9 51.3-114.9 114.9s51.3 114.9 114.9 114.9S339 319.5 339 255.9 287.7 141 224.1 141zm0 189.6c-41.1 0-74.7-33.5-74.7-74.7s33.5-74.7 74.7-74.7 74.7 33.5 74.7 74.7-33.6 74.7-74.7 74.7zm146.4-194.3c0 14.9-12 26.8-26.8 26.8-14.9 0-26.8-12-26.8-26.8s12-26.8 26.8-26.8 26.8 12 26.8 26.8zm76.1 27.2c-1.7-35.9-9.9-67.7-36.2-93.9-26.2-26.2-58-34.4-93.9-36.2-37-2.1-147.9-2.1-184.9 0-35.8 1.7-67.6 9.9-93.9 36.1s-34.4 58-36.2 93.9c-2.1 37-2.1 147.9 0 184.9 1.7 35.9 9.9 67.7 36.2 93.9s58 34.4 93.9 36.2c37 2.1 147.9 2.1 184.9 0 35.9-1.7 67.7-9.9 93.9-36.2 26.2-26.2 34.4-58 36.2-93.9 2.1-37 2.1-147.8 0-184.8zM398.8 388c-7.8 19.6-22.9 34.7-42.6 42.6-29.5 11.7-99.5 9-132.1 9s-102.7 2.6-132.1-9c-19.6-7.8-34.7-22.9-42.6-42.6-11.7-29.5-9-99.5-9-132.1s-2.6-102.7 9-132.1c7.8-19.6 22.9-34.7 42.6-42.6 29.5-11.7 99.5-9 132.1-9s102.7-2.6 132.1 9c19.6 7.8 34.7 22.9 42.6 42.6 11.7 29.5 9 99.5 9 132.1s2.7 102.7-9 132.1z"
              />
            </svg>
          </a>
        </div>
      </div>
      <div
        className="md:hidden lg:block"
      >
        <h3
          className="text-2xl font-medium text-dark capitalize drop-shadow-lg"
        >
          Candidate
        </h3>
        <ul
          className="flex flex-col gap-1 mt-5 text-gray"
        >
          <li>
            <a
              className="duration-300 hover:text-green"
              data-testid="candidate-link"
              href="/"
              onClick={[Function]}
            >
              Browse Jobs
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/"
              onClick={[Function]}
            >
              Salary Tools
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/"
              onClick={[Function]}
            >
              Career Advice
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/"
              onClick={[Function]}
            >
              Student Career Center
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/"
              onClick={[Function]}
            >
              Resume writing services
            </a>
          </li>
        </ul>
      </div>
      <div
        className="hidden md:block lg:hidden"
      >
        <h3
          className="text-2xl font-medium text-dark capitalize drop-shadow-lg"
        >
          Contact Us
        </h3>
        <div
          className="text-gray"
        >
          <p
            className="mt-5"
          >
            43/7 Northern Tower, Banani, Dhaka-1263, Bangladesh
          </p>
          <p
            className="mt-4 lowercase"
          >
            support@hirewave.com
          </p>
          <p>
            +8801543653473
          </p>
        </div>
      </div>
      <div>
        <h3
          className="text-2xl font-medium text-dark capitalize drop-shadow-lg"
        >
          Recruiters
        </h3>
        <ul
          className="flex flex-col gap-1 mt-5 text-gray"
        >
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/"
              onClick={[Function]}
            >
              How to Hire
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/"
              onClick={[Function]}
            >
              Talent Marketplace
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/"
              onClick={[Function]}
            >
              Hire an Agency
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/"
              onClick={[Function]}
            >
              Hire worldwide
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/"
              onClick={[Function]}
            >
              Direct Contracts
            </a>
          </li>
        </ul>
      </div>
      <div
        className="order-none"
      >
        <h3
          className="text-2xl font-medium capitalize text-dark drop-shadow-lg"
        >
          Resources
        </h3>
        <ul
          className="flex flex-col gap-1 mt-5 text-gray"
        >
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/"
              onClick={[Function]}
            >
              Help and Support
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/pricing"
              onClick={[Function]}
            >
              Pricing
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/events"
              onClick={[Function]}
            >
              Event
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/learning"
              onClick={[Function]}
            >
              Learning
            </a>
          </li>
          <li>
            <a
              className="duration-300 hover:text-green"
              href="/contact"
              onClick={[Function]}
            >
              Contact
            </a>
          </li>
        </ul>
      </div>
      <div
        className="md:hidden lg:block"
      >
        <h3
          className="text-2xl font-medium text-dark capitalize drop-shadow-lg"
        >
          Contact Us
        </h3>
        <div
          className="text-gray"
        >
          <p
            className="mt-5"
          >
            43/7 Northern Tower, Banani, Dhaka-1263, Bangladesh
          </p>
          <p
            className="mt-4 lowercase"
          >
            support@hirewave.com
          </p>
          <p>
            +8801543653473
          </p>
        </div>
      </div>
      <div
        className="hidden md:block lg:hidden"
      >
        <h3
          className="text-2xl font-medium text-dark capitalize drop-shadow-lg"
        >
          Job seekers
        </h3>
        <ul
          className="flex flex-col gap-1 mt-5 text-gray"
        >
          <li>
            <a
              className="hover:text-green duration-300"
              href="/"
              onClick={[Function]}
            >
              Browse Jobs
            </a>
          </li>
          <li>
            <a
              className="hover:text-green duration-300"
              href="/"
              onClick={[Function]}
            >
              Salary Tools
            </a>
          </li>
          <li>
            <a
              className="hover:text-green duration-300"
              href="/"
              onClick={[Function]}
            >
              Career Advice
            </a>
          </li>
          <li>
            <a
              className="hover:text-green duration-300"
              href="/"
              onClick={[Function]}
            >
              Student Career Center
            </a>
          </li>
          <li>
            <a
              className="hover:text-green duration-300"
              href="/"
              onClick={[Function]}
            >
              Resume writing services
            </a>
          </li>
        </ul>
      </div>
    </div>
  </div>
  <div
    className="border-t border-green mt-14 shadow-3xl"
  >
    <div
      className="container"
    >
      <div
        className="flex flex-col items-center justify-between py-5 md:flex-row"
      >
        <p>
          © 2023 Hire Wave. All rights reserved.
        </p>
        <div
          className="flex items-center gap-1"
        >
          <a
            className="duration-300 hover:text-green"
            href="/"
            onClick={[Function]}
          >
            Privacy Policy
          </a>
           
          |
          <a
            className="duration-300 hover:text-green"
            href="/"
            onClick={[Function]}
          >
            Terms of Services
          </a>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
